src/components/Player.js

```diff
@@ -11,27 +11,31 @@ function Player(props) {
     props.onEnd()
   }
 
-  return(<ReactPlayer
-    url={`https://www.youtube.com/watch?v=${props.videoId}`}
-    controls={true}
-    loop={props.repeatVideo}
-    onReady={(event) => onReady(event)}
-    onEnded={() => props.onEnd()}
-    onError={() => onError()}
-    playing={true}
-    width={"100%"}
-    height={"800px"}
-    config={{
-      youtube: {
-        playerVars: {},
-        embedOptions: {
-          host: "https://www.youtube-nocookie.com",
-        }
-      },
-      attributes: {},
-      tracks: [],
-    }}
-  />);
+  return(
+    <div style={{display: props.hideVideo ? 'none' : 'block'}}>
```

Comment:
I'm surprised that `ReactPlayer` doesn't have a prop available to do this tbh.
Maybe we can open a PR up for them ;)